contrib/custom_top_level_templates.go

```diff
@@ -96,10 +98,12 @@ const (
 // Returns the template for generating input snapshots.
 func InputSnapshot(params SnapshotTemplateParameters) model.CustomTemplates {
 	templateContentsBytes, err := ioutil.ReadFile(templatesDir + InputSnapshotCustomTemplatePath)
+	var templateContents string
 	if err != nil {
-		panic(err)
+		contextutils.LoggerFrom(nil).DPanic(err)
+		templateContents = ""
```

Comment:
This sets the templatecontents, and then the next line (outside the conditional) overwrite it. Do we need to do this or should we be returning an error here?